### DISCUSSION TITLE
Ping CRM - World's First Inertia (demo) Application

### DISCUSSION THREAD

**Discussion by claudiodekker** [inertiajs/inertia#360] [Show and tell]
To kick things off, as well as to provide a good example, we've setup a demo app for Inertia.js, called [Ping CRM](http://demo.inertiajs.com/). 

This application is built using Laravel and Vue.js, and the source code is available [here](https://github.com/inertiajs/pingcrm).

![image](https://user-images.githubusercontent.com/1752195/102696954-78f43380-4232-11eb-826c-cc6c52f247e5.png)

Most if not all of the approaches taken in the Ping CRM application are what we consider to be Inertia.js 'best practices'. 
This alone, apart from our documentation, makes it a very good source of information!

Enjoy!

**Comment by reinink** [comment]
You can also find numerous 3rd party Ping CRM implementations using other languages and frameworks here: https://inertiajs.com/demo-application
